README.md

```diff
@@ -1,2 +1,52 @@
-# task1
-Change
+<h1 align="center">
+  <br>
+  <img src="" width="200">
+  <br>
+  WeatherAPI
+  <br>
+</h1>
+
+<h4 align="center">Simple Weather API built in GO language.</h4>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#credits">Credits</a>
+</p>
+
+## Key Features
+
+* Simplified Information In JSON format
+    - <img src="" width="200">
+* Without Complex Integration
+    - Input desired town
+* Returned Error
+    - If it can't get information about weather it returns appropriate error
+  
+
+## How To Use
+
+To clone and run this application, you'll need [GIN](https://github.com/gin-gonic/gin) and [GO](https://go.dev/dl/) installed on your computer. From your command line:
```

Comment:
There's a lot more dependencies there than just gin. If you need to mention it, rather say it's built using gin framework.
Be exact in this kind of things!